tests/basic_test.cpp

```diff
@@ -373,39 +373,6 @@ BOOST_AUTO_TEST_CASE(exception_in_startup)
    app_thread.join();
 }
 
-// -----------------------------------------------------------------------------
```

Comment:
Is it possible to add a test case for the restored behavior so that it won't regress in the future?

Reply:
Hey Lin, I added the test, thanks!